packages/framework/src/Form/FileUploadType.php

```diff
@@ -44,12 +46,15 @@ public function __construct(FileUpload $fileUpload)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults([
+        $resolver
+            ->setDefaults([
             'error_bubbling' => false,
             'compound' => true,
             'file_constraints' => [],
             'multiple' => false,
-        ]);
+            'image_entity' => null,
```

Comment:
image_entity should either not be part of this commit, or it should be mentioned in the commit msg